keyboards/keebio/bdn9/keymaps/vosechu-browser/keymap.c

```diff
@@ -0,0 +1,72 @@
+#include QMK_KEYBOARD_H
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define _LAYER_ KC_TRNS
+#define XXXXXXX KC_NO
+
+// Macro keys for some apps
+#define SLACKUP LALT(LSFT(KC_UP))
+#define SLACKDN LALT(LSFT(KC_DOWN))
+#define RELOAD  LGUI(KC_R)
+
+enum {
+  PAWFIVE = SAFE_RANGE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(
+    RESET  , PAWFIVE, RELOAD , \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".